components/hybrid_engine/src/write_batch.rs

```diff
@@ -130,20 +130,26 @@ impl<EK: KvEngine> Mutable for HybridEngineWriteBatch<EK> {
     }
 
     fn delete_range(&mut self, begin_key: &[u8], end_key: &[u8]) -> Result<()> {
-        self.disk_write_batch.delete_range(begin_key, end_key)
+        self.disk_write_batch.delete_range(begin_key, end_key)?;
+        self.cache_write_batch.delete_range(begin_key, end_key)
```

Comment:
Looks like they are not in atomic operation, does it cause data inconsistency?

Reply:
delete_range in in-memory engine is to evict the range rather than delete data directly. So this operation itself will not cause data inconsistency as evict range can happen at anytime.

Reply:
I see, could you add some comments?